packages/apps/plugins/plugin-ipfs/src/components/FileMain.tsx

```diff
@@ -59,8 +59,8 @@ export const FileSlide: FC<{ file: TypedObject }> = ({ file }) => {
 
   // TODO(burdon): Config object-container/-fit.
   return (
-    <div className='h-full'>
-      <FilePreview type={file.type} url={url} className='object-cover' />
+    <div className='h-full flex justify-center align-center'>
```

Comment:
perhaps set `cover` as an option so that both are supported in principle and you can decide on the default.
then later we can figure out how the user can configure this (e.g., markdown frontmatter, or some UX chrome in the slide section).

Reply:
were you thinking passing a `cover` parameter to the `FileSlide` component?

Something like this:

```tsx
export const FileSlide: FC<{ file: TypedObject; cover?: boolean }> = ({ file, cover }) => {
  const config = useConfig();
  if (!file.cid) {
    return null;
  }

  const url = getIpfsUrl(config, file.cid);

  // TODO(burdon): Config object-container/-fit.
  return (
    <div className='h-full flex justify-center align-center'>
      <FilePreview type={file.type} url={url} className={`object-${cover ? 'cover' : 'contain'}`} />
    </div>
  );
};
```